packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -0,0 +1,693 @@
+# Cart and wishlist
+
+Customer's cart and wishlist can be managed using `useCart` and `useWishlist` composables respectively, provided by every integration. Data can be accessed using `cartGetters` and `wishlistGetters`. 
+
+## Loading and creating the cart
+
+The `load` function will load your cart from the server or create a new one if it doesn't exist. The `cart` object will be `null` until you load it.
+
+
+```vue
+<script>
+  import { useCart } from '{INTEGRATION}';
+  import { onSSR } from '@vue-storefront/core';
+  export default {
+    setup() {
+      const {
+        cart,
+        load
+      } = useCart();
+
+      onSSR(async () => {
+        await load();
+      });
+
+      return {
+        cart,
+        load
+      };
+    }
+  };   
+</script> 
+```
+
+
+## Adding an item to the cart
+
+To add the product to the cart you can use `addItem` method:
+
+```vue
+<template>
+  // ...
+    <ul>
+      <li
+        v-for="(product, i) in products" :key="i"
+      > 
+        // ...
+        <button
+          @click="addItem({ product, quantity })"
+        >
+          Add to cart
+        </button>
+      </li>
+    </ul>
+  // ...
+</template>    
+<script>     
+  import { computed } from '@vue/composition-api';
+  import { useCart } from '{INTEGRATION}';
+  export default {
+    props: {
+      products: {
+        type: Array,
+        required: true
+      }
+    },
+    setup() {
+      const {
+        addItem,
+      } = useCart();
+
+      return {
+        addItem,
+      };
+    }
+  };
+</script>
+``` 
+
+## Removing items and changing their quantity
+
+To remove an item from the cart use `removeItem` function, and similarly to update quantity use `updateItemQty` function:
+
+```vue
+<template>
+  // ...
+    <div>
+      <ul>
+        <li
+          v-for="(product, i) in products" :key="i"
```

Comment:
**nitpick**: could we use `product.id` as a key here? Using index is not optimal and we don't recommend it on Vue 3 docs. Total nitpick, please feel free to ignore 😅